### DISCUSSION TITLE
Using 2F1 from two different computers

### DISCUSSION THREAD

**Discussion by StefThomas** [community/community#66306] [New to GitHub]
Hi,

I need to access my GitHub account  from two different computers (even three actually), I can’t see how to configure 2FA from both.

**Comment by queenofcorgis** [comment]
Hi there, 

There are plenty of options for your 2FA setup - I think for your use the GitHub mobile app may be a great choice for a second factor - but check out our other options [here](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication)!
